test/screenshot/run-upload-assets.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const assetUploader = require('./asset-uploader');
+
+assetUploader.upload()
+  .then(
+    (files) => {
+      const htmlFileUrls =
+        files
+          .filter((file) => file.relativePath.endsWith('.html'))
+          .map((file) => file.fullUrl)
+          .sort();
+      console.log('\n\nDONE!\n\n');
+      console.log(htmlFileUrls.join('\n'));
+    },
+    (err) => {
```

Comment:
Nit: Use `.catch` instead. It's 100% a personal preference but I find it much more obvious what the function is doing.

Reply:
Done